### ISSUE TITLE
Oled 1.3

### ISSUE THREAD

**Issue by carl1961** [Apehaenger/Another-HotPlate-Firmware#2]
Awesome program Thanks!
As  it is completely usable on 1.3 OLED
would it be possible to change to allow 1.3 OLED. 
I tried changing in Ui.hpp   , caused lots compile errors

//U8G2_SSD1306_128X64_NONAME_1_HW_I2C u8g2;
U8GLIB_SH1106_128X64 u8g(U8G_I2C_OPT_NONE); // U8glib\examples\Menu\Menu.ino

![IMG_20230511_051802677](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/3c95619b-850d-4390-b4e2-944e89e744c6)
![IMG_20230511_052328098](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/53e60131-7aed-412a-9982-af5ee66e4c38)

**Comment by carl1961** [comment]
One thing I use ardunio lib example,  I see lib in platformio has different examples, may should use them.

this is back of 1.3 OLED
![71-3bW84UrL _AC_SL1500_](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/96949540-312a-4742-92e0-61b1dd90480d)
![71gwqALW5PL _AC_SL1500_](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/3237c166-28aa-4772-83a2-826af5aec9d7)

**Comment by Apehaenger** [comment]
Hi Carl!

I compared it with mine, but unfortunately mine is  completely different.

If I understood your initial message right, than you got some kind of sample running with this initialization line:
```U8GLIB_SH1106_128X64 u8g(U8G_I2C_OPT_NONE); // U8glib\examples\Menu\Menu.ino``` ?

If so, we know at least that your OLED has an SH1106 controller.

Please try one of the changes of the following commit (the first already uncommented one may work already): 70c21ab6d5b30f9f0d2ad6b3fab94c7013d9d3bb

If none of them work, I need some more information about your OLED model. I.e. link where you ordered it ;-)

**Comment by carl1961** [comment]
Thanks, That worked perfect, all ok, just what you had already uncommented
![Worked](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/5bc98676-015e-419c-b0a8-279ce00f3f33)

**Comment by Apehaenger** [comment]
Quite cool!

What was/is wrong with the stock-controller?

Send me a link once you're ready. If you make it good readable I probably link it within my readme ;-)

**Comment by carl1961** [comment]
original controller is cheap on off temp 
![IMG_20230512_171437705](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/47c1107e-340e-419e-9211-01188b5a115e)
![IMG_20230512_171420659](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/68adc3d7-72d7-4eb3-afc2-f0a0e6241dda)
![IMG_20230512_171411001](https://github.com/Apehaenger/Another-HotPlate-Firmware/assets/3056821/a52b69ce-aeb3-4037-9285-d4d5fbdfd439)

**Comment by Apehaenger** [comment]
Ahhh, and doesn't work reliable?
I ask because configuring the PID-Tuner of my FW might become somehow... time-consuming?
Or is it the reflow-profile in which you're interested?

**Comment by carl1961** [comment]
I have not used it yet. took it all apart and did a cad file of the parts. I watched many YouTube videos on these hot plates. yes want to use your automated fw . this hot plate came with 4 6mm x 80mm Cartridge Heater, every where I found the Cartridge Heater online shows them to be 120 watts, so most this hot plate can do is 480 watts. don't see how it could be advertised as 850 watts.  model of hotplate is 946C+ .
